spec/support/fake_ahoy_tracker.rb

```diff
@@ -0,0 +1,5 @@
+class FakeAhoyTracker
```

Comment:
I am not opposed to this strategy, but it probably makes sense to have a test that confirms that `FakeAhoyTracker` and `Ahoy::Tracker` have the same interface. 

As is, I can imagine a scenario in which the ahoy lib requires new/different args for `initialize`, for example, but the specs still pass without a change. Am I making sense?